app/services/green_lanes/determine_next_page.rb

```diff
@@ -65,7 +65,7 @@ def question_unanswered?(cat_exemptions_apply)
     def new_exemptions_path(category, params = {})
       query = params.to_query
 
-      "/green_lanes/applicable_exemptions/new?category=#{category}" + (query.present? ? "&#{query}" : '')
+      "/check-spimm-eligibility/applicable_exemptions/new?category=#{category}" + (query.present? ? "&#{query}" : '')
```

Comment:
Could we use the path helper?

Reply:
sure, I've also used url_helpers inside the DetermineNextPage,
much better now.